config/config.inc

```diff
@@ -115,8 +115,51 @@ if [ -n "$FF_MAX_DIM" ]; then
   SET_MAX_DIM="-DFF_MAX_DIM=${FF_MAX_DIM}"
 fi
 
+# set ROCM path
+if [ -n "$ROCM_PATH" ]; then
```

Comment:
```suggestion
if [[ -n "$ROCM_PATH" ]]; then
```

Reply:
Other conditionals in script use single bracket conditionals -- should we just keep it consistent?

Reply:
Yeah match the style of the script